### PR TITLE
cluster-autoscaler

### PR DESCRIPTION
**Pull request by resmaa**
**3.1 Release Feature**

Cluster aAutoscaler
PCP-630

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1052#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1019]: This does not need to be in an info box, it should be the heading's introductory paragraph.

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1018]:
```suggestion
## Edit Node Pool

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1029]:
```suggestion
|**Allow worker capability **|Select this option for allowing workloads to be provisioned on master nodes.|
```
We are already in the context of Master Pool, so we don't need to specify master pool.

Why do we need to

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1030]:
```suggestion
|**[Labels](/clusters/cluster-management/taints#overviewonlabels)**| Add a label to apply placement constraints on a pod, such as a node eligible for receiving the workload. 

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1033]:
```suggestion
|**Availability Zones**| Choose one or more availability zones. Palette provides fault tolerance to guard against hardware failures, network failures, etc., by provisioning nodes across availability zones if multiple zones are selected.|
```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1041]:
```suggestion
|**Enable Autoscaler**|You can enable the autoscaler, by toggling the **Enable Autoscaler** button. Autoscaler scales up and down resources between the defined minimum and the maximum number of nodes to optimize resource utilization.|
```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1047]:
```suggestion
|**[Labels](/clusters/cluster-management/taints#overviewonlabels)**|Add a label to apply placement constraints on a pod, such as a node eligible for receiving the workload.

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1050]:
```suggestion
|**Availability Zones**| Choose one or more availability zones. Palette provides fault tolerance to guard against hardware failures, network failures, etc., by provisioning nodes across availability zones if multiple zones are selected.|
```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:1051]:
```suggestion
|**Disk Size**|Provide the required storage size|

```

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:110]:
```suggestion
    |**[Labels](/clusters/cluster-management/taints#overviewonlabels)**|Add a label to apply placement constraints on a pod, such as a node eligible for receiving the workload.

```

File: content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md:127]:
```suggestion
    |**[Labels](/clusters/cluster-management/taints#overviewonlabels)**|Add a label to apply placement constraints on a pod, such as a node eligible for receiving the workload.

```

File: content/docs/04-clusters/01-public-cloud/05-gcp.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/05-gcp.md:220]:
```suggestion
  |**Enable Autoscaler**|You can enable the autoscaler, by toggling the **Enable Autoscaler** button. Autoscaler scales up and down resources between the defined minimum and the maximum number of nodes to optimize resource utilization.|
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and suggestions below.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1052#pullrequestreview-1225122931" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1052#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by karl-cardenas-coding** [comment]
@resmaa, I'm gonna close this PR. I went ahead and carried over the changes in a new PR #1061 that is based of the release branch. That gets us around the merge conflicts.

**Pull request closed without merging**
